Content.Client/_RMC14/Xenonids/Fruit/XenoFruitChooseSystem.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Content.Shared._RMC14.Xenonids.Fruit;
+using Content.Shared._RMC14.Xenonids.Fruit.Components;
+using Robust.Shared.Timing;
+
+namespace Content.Client._RMC14.Xenonids.Fruit;
+
+public sealed class XenoFruitChooseSystem : EntitySystem
+{
+    [Dependency] private readonly IGameTiming _timing = default!;
+
+    public override void Initialize()
+    {
+        SubscribeLocalEvent<XenoFruitPlanterComponent, AfterAutoHandleStateEvent>(OnXenoFruitAfterState);
+    }
+
+    private void OnXenoFruitAfterState(Entity<XenoFruitPlanterComponent> ent, ref AfterAutoHandleStateEvent args)
+    {
+        if (!_timing.IsFirstTimePredicted)
+            return;
+
+        if (!TryComp(ent, out UserInterfaceComponent? ui))
+            return;
+
+        foreach (var bui in ui.ClientOpenInterfaces.Values)
+        {
+            if (bui is XenoFruitChooseBui chooseUi)
+                chooseUi.Refresh();
+        }
```

Comment:
This should be wrapped in a try catch with a log.error inside the catch in case an error happens so you don't brick the client
OverwatchConsoleSystem on the client should have an example of this

Reply:
Added a try-catch block around this part of the code